css-ecommerce-page/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <section>
+      <div class="row">
+        <div class="columnfull">
+          <h1>New Arrivals</h1>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="row">
+        <div class="columnfifth">
```

Comment:
One suggestion is to rename class from `columnfifth` to `column-fifth` or `col-fifth`. It helps to make class names more readable. No need to change for this exercise but something to consider!